src/LaunchDarkly/LDClient.php

```diff
@@ -46,9 +48,14 @@ public function __construct($apiKey, $options = []) {
             $options['capacity'] = 1000;
         }
 
-        $this->_eventProcessor = new \LaunchDarkly\EventProcessor($apiKey, $options);
+        $this->_eventProcessor = new EventProcessor($apiKey, $options);
 
-        $this->_client = $this->_make_client($options);
+        if (isset($options['feature_requester_class'])) {
```

Comment:
A bit weird that the ctor parameters end up being fixed. Other feature requester impls won't care about the base URI, or API key, for example. I think it would be clearer to pass in the feature requester instance, rather that instantiating one here.